node/bft/src/helpers/storage.rs

```diff
@@ -1131,6 +1131,213 @@ pub(crate) mod tests {
         // Check that the underlying storage representation remains unchanged.
         assert_storage(&storage, &rounds, &certificates, &batch_ids, &transmissions);
     }
+
+    /// Test that `check_incoming_certificate` does not reject a valid cert.
+    #[test]
+    fn test_valid_incoming_certificate() {
+        let rng = &mut TestRng::default();
+
+        // Sample a committee.
+        let (committee, private_keys) =
+            snarkvm::ledger::committee::test_helpers::sample_committee_and_keys_for_round(1, 5, rng);
```

Comment:
This is a minor technicality that may not have any impact, but since we recently clarified the meaning of the starting_round of a committee as the one where the committee is created, we should pass 0 instead of 1 to this sampling function.